assets/queries/ansible/azure/azure_instance_using_basic_authentication/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "e2d834b7-8b25-4935-af53-4a60668dcbe0",
+  "queryName": "Azure Instance Using Basic Authentication",
+  "severity": "HIGH",
+  "category": "Best Practices",
+  "descriptionText": "Azure Instances should use SSH Key instead of basic authentication",
+  "descriptionUrl": "https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/linux_virtual_machine#admin_ssh_key",
```

```suggestion
  "descriptionUrl": "https://docs.ansible.com/ansible/latest/collections/azure/azcollection/azure_rm_virtualmachine_module.html#parameter-linux_config/disable_password_authentication",
```
Update the descriptioURL to the alsible docs